File: back/ProEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Descricao { get; set; }
        public DateTime DataEvento { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedeSocials { get; set; }
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }

        public Evento()
        {
            
        }
    }
}

File: front/ProEventos-App/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento {

     id: number;
     nome:string;
     descricao:string;
     dataevento?: Date;
     lotes: Lote[];
     redesocials: RedeSocial[];
     palestranteseventos: Palestrante[];
}
